code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -91,3 +91,15 @@
 	name = "snow strider meat"
 	desc = "A slab of nav'twir's meat, an animal native from Adhomai."
 	icon_state = "adhomai_meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poor, eating the poor mice of the station."
+
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice/Initialize()
+	. = ..()
+	reagents.add_reagent("protein", 5)
+	reagents.add_reagent("toxin", 2)

```

Comment:
Why exactly are mice poisonous? Considering historically speaking people ate mice without ill effect, I don't really see the intent here. If you meant to make them less nutritous, then you should use the bad nutrition that junk food has, although even that is questionable.

Reply:
I said that earlier and they ignored me then. Maybe they'll listen this time when you say it.

Reply:
Mice in a survival situation are abit more safer to eat then IE, a factory worker stabbing a mouse and cooking ti and eating it. The difference is the station is a industrial enviroment and mice as there nature crawl arouind and are exposed to harmful enviroments and diseases.

Reply:
If you want to make the mice disease to figure out a way to give him a disease that can be transmitted through meat. Don't just arbitrarily make their meat toxic. This is a really clumsy implementation of what might be a good idea and needs Improvement if you really want to do it.